fendermint/vm/message/src/ipc.rs

```diff
@@ -25,8 +25,9 @@ pub enum IpcMessage {
     /// or we can gossip votes using the _IPLD Resolver_ and attach them as a quorum certificate.
     BottomUpExec(CertifiedMessage<BottomUpCheckpoint>),
 
-    // TODO
-    TopDown,
+    /// A top-down checkpoint parent finality proposal. This proposal should contain the latest parent
+    /// state that to be checked and voted by validators.
+    TopDownProposal(ParentFinalityProposal),
```

Comment:
```suggestion
    TopDownExec(ParentFinalityProposal),
```

To be symmetric with `BottomUpExec`, we are proposing that this thing gets executed because it is final.